### DISCUSSION TITLE
Have output in different folder.

### DISCUSSION THREAD

**Discussion by zemega** [quarto-dev/quarto-cli#1046] [Q&A]
Hello. Is it possible to have Quatro output a word document in a different folder? Previously in RMarkdown and Knitr, I used the option below to have the output go up one path, enter a Documents folder, and have the output there.
```
---
knit: (function(inputFile, encoding) {
  rmarkdown::render(inputFile, encoding = encoding, output_dir = "../Documents") })
---
```

**Comment by jjallaire** [comment]
If you create a project then you can have all output go into an `output-dir`: https://quarto.org/docs/projects/quarto-projects.html

**Reply by zemega** [reply]
> If you create a project then you can have all output go into an `output-dir`: https://quarto.org/docs/projects/quarto-projects.html

That sounds good for a project. What if I want to have several output directory for different documents? Such as below. For a single project, I'm using different programs for each part of the project.
1. `../` for general project documentation/introduction
2. `../McIDAS-V` for tutorial on how to use this program for this project that takes its data from `../Data`
3. `../Document` for reports and work logs and such

I personally feels that this should be a basic option.
